hazelcast/src/test/java/com/hazelcast/scheduledexecutor/ScheduledExecutorServiceSlowTest.java

```diff
@@ -370,4 +373,27 @@ public void cancelUninterruptedTask_waitUntilRunCompleted_killMember_checkStatus
         assertTrue(future.isDone());
     }
 
+    @Test
+    public void reschedulingAfterMigration_whenCurrentNodePreviouslyOwnedTask() {
+        TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory();
+        HazelcastInstance first = factory.newHazelcastInstance();
+
+        int tasksCount = 1000;
+        final IScheduledExecutorService scheduler = getScheduledExecutor(new HazelcastInstance[] {first }, "scheduler");
```

Comment:
I thought there would be more logic inside the `getScheduledExecutor()` method, but since it isn't I would just use `first.getScheduledExecutorService("scheduler");`